db/migrate/20161128200400_update_quotes.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateQuotes < ActiveRecord::Migration
+  def change
+    change_column_null :quotes, :anime_id, false
+    change_column_null :quotes, :user_id, false
+    change_column_null :quotes, :content, false
+
+    add_foreign_key :quotes, :users
+    add_foreign_key :quotes, :anime
+    add_foreign_key :quotes, :characters
```

Comment:
I generally like to split the `add_column` from the backfill of `Anime` value, because using `default: 'Anime'` sets an actual database default... which could cause silent and confusing errors down the road if we accidentally use media_id instead of media

I'd also advise moving the destruction down by `change_column_null` just to have a narrower window of fuckupery

The rest seems pretty much spot-on